src/main.ts

```diff
@@ -35,15 +37,21 @@ void pipe(
   TE.bind("prisma", (env) =>
```

Comment:
We should remove everything that is prisma and mongo related.

Reply:
I removed almost everything, the only thing left is to remove prisma and mongo from dockerfile and docker compose and add cosmos